### DISCUSSION TITLE
Run using Docker Toolbox on Windows

### DISCUSSION THREAD

**Discussion by awirthy** [Tzahi12345/YoutubeDL-Material#394] [Q&A] [answered]
I'm running the following command to install into Docker Toolbox on Windows 7.
Any ideas on why I'm getting the 'invalid reference format: repository name must be lowercase' error?

Code:
`docker run -d --name youtubedlmaterial -v $(PWD)/youtubematerial/data:/app:appdata -v $(PWD)/youtubematerial/audio:/app/audio -v $(PWD)/youtubematerial/video:/app/video -v $(PWD)/youtubematerial/subscriptions:/app/subscriptions -v $(PWD)/youtubematerial/users:/app/users:/app/users -p 8998:17442 tzahi12345/youtubedl-material`

Any help is appreciated.

**Comment by Tzahi12345** [comment, marked as answer]
Have you tried the docker-compose method? This should be a lot easier, just grab it from the [latest release](https://github.com/Tzahi12345/YoutubeDL-Material/releases/tag/v4.2) and run `docker-compose up`

**Reply by awirthy** [reply]
Yes. I did manage to get it working by doing that.

**Reply by Tzahi12345** [reply]
Awesome, great to hear
